explainability-connectors/src/test/java/org/kie/trustyai/connectors/kserve/v2/KServeFormatsTest.java

```diff
@@ -0,0 +1,742 @@
+package org.kie.trustyai.connectors.kserve.v2;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.kie.trustyai.connectors.kserve.v2.grpc.InferTensorContents;
+import org.kie.trustyai.connectors.kserve.v2.grpc.ModelInferRequest;
+import org.kie.trustyai.connectors.kserve.v2.grpc.ModelInferResponse;
+import org.kie.trustyai.explainability.model.*;
+import org.kie.trustyai.explainability.model.TensorDataframe;
+
+import com.google.protobuf.ByteString;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.kie.trustyai.connectors.kserve.v2.TensorConverter.parseKserveModelInferRequest;
+import static org.kie.trustyai.connectors.kserve.v2.TensorConverter.parseKserveModelInferResponse;
+
+public class KServeFormatsTest {
+
+    private static final String MODEL_ID = "example-1";
+    private static final String MODEL_VERSION = "0.0.1";
```

Comment:
Let's use a different value to spot any other possible place where the version was hard coded 
```suggestion
    private static final String MODEL_VERSION = "0.0.1-TEST";
```